spec/18-namespaces.md

```diff
@@ -121,6 +121,8 @@ namespace NS3\Sub1;
 <pre>
   <i>namespace-use-declaration:</i>
     use  <i>namespace-function-or-const<sub>opt</sub></i> <i>namespace-use-clauses</i>  ;
+    use  <i>namespace-function-or-const<sub>opt</sub></i> <i>namespace-name-as-a-prefix</i>
```

Comment:
Instead of `namespace-name-as-a-prefix` this should be `\_opt   namespace-name    \`. The former would include empty NS and namespace-relative.